src/ports/catalog/queries.ts

```diff
@@ -701,7 +716,7 @@ export const getCollectionsItemsCatalogQueryWithTrades = (filters: CatalogQueryF
       .append(getCollectionsQueryWhere(filters, true))
   )
 
-  addQuerySort(query, filters)
+  addQuerySort(query, filters, true)
```

Comment:
should we use here a feature flag?

Reply:
the feature flag is configured in the Front, it either using v1 or v2 based on the FF, so this `true` would be for V2. Does it make sense?

Reply:
yeah, it makes sense!